exporter/type.go

```diff
@@ -44,6 +44,9 @@ type (
 )
 
 func (s *StaticConfig) Validate() error {
+	if len(s.Clusters) == 0 {
+		return fmt.Errorf("at least one cluster in config")
```

Comment:
Thanks a lot!

How about minor rephrasing as:

```diff
- 		return fmt.Errorf("at least one cluster in config")
+		return fmt.Errorf("At least one cluster should be configured.")
```

Reply:
ok. But the capitalized letters and punctuation marks are removed as the Golang style guide recommends.

https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Sorry, am not a gopher, please ignore the suggestion on the Big A part.